doc/release/RELEASE-PROTOCOL.md

```diff
@@ -42,6 +42,7 @@ This document outlines the SDLC pillars of the opensupplyhub monorepo, as well a
 | v1.17.0 | July 27, 2024  | July 23, 2024 | @Roman Stolar |
 | v1.18.0 | August 10, 2024  | August 6, 2024 | @Vlad Shapik |
 | v1.19.0 | August 24, 2024  | August 20, 2024 | @Vlad Shapik |
+| v1.20.0 | September 07, 2024  | September 03, 2024 | @Anhelina Bohdanova |
```

Comment:
I think it make sense you to handle release twice:

The first you get knowledges and understanding how the process work
The second you'll try to follow it on your own

Reply:
Assigned myself for second week